src/antidote_crdt_generic.erl

```diff
@@ -18,6 +18,8 @@
 -define(newjavaid, 4).
 -define(loadsnapshot, 5).
 
+-define(backendtimeout, 5000).
+
```

Comment:
do we want to revert this or leave it hilarious?

Reply:
Apparently by "revert this" I mean "set it less high" but also that's fine to keep it high for a while.

Reply:
Well 5000 milliseconds is 5 seconds. It isn't ~that~ crazy and I think it is what we had before. I do 5000000 when I am debugging